src/osvr/Server/ConfigureServerFromFile.cpp

```diff
@@ -0,0 +1,181 @@
+/** @file
+    @brief Header
+
+    @date 2014
+
+    @author
+    Sensics, Inc.
+    <http://sensics.com/osvr>
+*/
+
+// Copyright 2014 Sensics, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Internal Includes
+#include <osvr/Server/Server.h>
+#include <osvr/Server/ConfigureServer.h>
+#include <osvr/Server/ConfigureServerFromFile.h>
+#include <osvr/Server/ConfigFilePaths.h>
+#include <osvr/Util/PlatformConfig.h>
+#include <osvr/Util/Logger.h>
+#include <osvr/Util/LogNames.h>
+
+// Library/third-party includes
+// - none
+
+// Standard includes
+#include <exception>
+#include <fstream>
+#include <iostream>
+#include <vector>
+#include <sstream>
+
+namespace osvr {
+namespace server {
+
+    ServerPtr configureServerFromString(std::string const &json) {
```

Comment:
Thanks for moving this to the `.cpp` file. I meant to do that when I was originally refactoring and it slipped my mind later.